alpine-nix-rails-nginx/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# dkubb/alpine-nix-rails-nginx
+
+FROM dkubb/alpine-nix:latest
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+EXPOSE 80
```

Comment:
> That's because we aren't using a VPC. Our network traffic can't be guaranteed by the infrastructure as being private. With a VPC the system is designed to segment the traffic from all other customers.

<devils advocate>
Before where we used `EC2 <-> ELB` our data traveled a "public" network. True. On that network we decided to not trust (as an example) the routers in between. So we set-up SSL between the backend and the ELB.

Now, the VPC feature is a router feature addon, why would we now trust the routers when we did not before?
</devils advocate>

As I said: I'm fine in skipping this for now and "trust" the VPC. Just giving counter arguments to explore the discussion.